<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Visualization</class>
 <widget class="QMainWindow" name="Visualization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualization App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="equationsTxt">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>77</y>
      <width>1001</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="equationsLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Equations</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="outputTxt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>1001</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="outputLbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>20</y>
      <width>251</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot Algebraic Curves</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Rm Pol/mial</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="rootsTxt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>1001</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="rootsLbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string> Roots (x , y)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>true</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuGnupoints">
    <property name="title">
     <string>Gnupoints</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionInsert"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGnupoints"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Alt+F</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="text">
    <string>Insert</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
